src/isValidRequest.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+'use strict';
+
+const allowedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function isValidRequest(text, toCase) {
+  const statusText = {
+    errors: [],
+  };
+
+  if (!text) {
+    statusText.errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    statusText.errors.push({
+      message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!(allowedCases.includes(toCase)) && toCase) {
```

Comment:
Check if you have a `toCase` value first.

```suggestion
  if (toCase && !allowedCases.includes(toCase)) {
```